microraft/src/main/java/io/microraft/impl/RaftNodeImpl.java

```diff
@@ -157,6 +157,9 @@ public final class RaftNodeImpl implements RaftNode {
     private final String localEndpointStr;
     private final Long2ObjectHashMap<OrderedFuture> futures = new Long2ObjectHashMap<>();
 
+    private final Random random;
+    private final Clock clock;
```

Comment:
We can also add a getter method for these fields and access them via those getters. Since the request and response handler classes all have access to RaftNodeImpl, we don't need to pass Clock and store it as a field in those classes and directly access it with `raftNode.getClock()`.  Let's apply this change and remove Clock c'tor param and member field from the handlers.

Reply:
Alright, this is done.